### DISCUSSION TITLE
Query params in json requests of SSG after next js upgrade to 12.1

### DISCUSSION THREAD

**Discussion by nagireddyy** [vercel/next.js#42216] [Help]
We have recently upgraded next js version from 11.xx to 12.1. We are using SSG build mechanisam in our web site.
Before the upgrade, when ever we request any page like www.abcxyz.com/product/redflowers?queryId=042d4d4751c82a9fa25af9a842c35209&indexName=default_ranking&category=flowers-summer, request for json file was like www.abcxyz.com/product/redflowers.json. Earlier query param was ignored when requesting json file.

But after upgradation, request for json file is like www.abcxyz.com/product/redflowers.json?queryId=042d4d4751c82a9fa25af9a842c35209&indexName=default_ranking&category=flowers-summer. Whatever the query params were part of the actual URL is appended to the json request.

These query params are dynamic and change everytime for the same url therefore json. 
Because of this, json file is not served from CDN cache, everytime request is going to actual server. So request times are increased and performance is impacted.

Is there any way to not to pass these query params for json requests. Can we set anything in next config or in getstaticprops method.

Thanks in advance.

**Comment by chrisneven** [comment]
Did you ever manage to find a solution @nagireddyy?
